src/Reflection/ReflectionProperty.php

```diff
@@ -12,30 +12,34 @@
  */
 class ReflectionProperty
 {
-    const IS_PRIVATE   = \ReflectionProperty::IS_PRIVATE;
```

Comment:
These properties were added intentionally so that our `ReflectionProperty` was compatible with the normal one.

Before considering to fix this: I'm not sure whether this class is still needed, since we upgraded the minimum php version a while ago.

Reply:
I could also just revert this for now and create an issue that we need to look into this.